flow/Makefile

```diff
@@ -355,12 +354,16 @@ versions.txt:
 # Create temporary Liberty files which have the proper dont_use properties set
 # For use with Yosys and ABC
 .SECONDEXPANSION:
-$(DONT_USE_LIBS): $$(filter %$$(@F) %$$(@F).gz,$(LIB_FILES))
+$(LIB_FILES_EXTRACTED): $$(filter %$$(@F) %$$(@F).gz,$(LIB_FILES))
 	@mkdir -p $(OBJECTS_DIR)/lib
-	$(UTILS_DIR)/preprocessLib.py -i $^ -o $@
+	@if echo "$<" | grep -q "\.gz$$"; then \
+		gunzip -c $< > $@; \
+	else \
+		cp $< $@; \
+	fi
```

Comment:
Does Yosys support reading in compressed .gz files and decompressing on-the-fly? This step could be entirely eliminated if so.

Reply:
It does not, but it could. It requires zlib by default to support .fst wave files. I'll see what I can do

Reply:
I think it does when you supply the file to a frontend (e.g. `read_liberty`) but not e.g. in `dfflibmap -liberty`

Reply:
It would be nice to support gz in both

Reply:
We're working on it